src/cephadm/cephadm

```diff
@@ -4643,6 +4643,50 @@ class Apt(Packager):
             logger.info('Podman did not work.  Falling back to docker...')
             self.install(['docker.io'])
 
+    def kubic_repo_url(self):
+        return 'https://download.opensuse.org/repositories/devel:/kubic:/' \
+               'libcontainers:/stable/xUbuntu_%s/' % self.distro_version
+
+    def kubic_repo_path(self):
+        return '/etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list'
+
+    def kubric_repo_gpgkey_url(self):
+        return '%s/Release.key' % self.kubic_repo_url()
+
+    def kubric_repo_gpgkey_path(self):
+        return '/etc/apt/trusted.gpg.d/kubic.release.gpg'
+
+    def add_kubic_repo(self):
+        url = self.kubric_repo_gpgkey_url()
+        logger.info('Installing repo GPG key from %s...' % url)
+        try:
+            response = urlopen(url)
+        except HTTPError as err:
+            logger.error('failed to fetch GPG repo key from %s: %s' % (
+                url, err))
+            raise Error('failed to fetch GPG key')
+        key = response.read().decode('utf-8')
+        tmp_key = write_tmp(key, 0, 0)
+        keyring = self.kubric_repo_gpgkey_path()
+        call_throws(['apt-key', '--keyring', keyring, 'add', tmp_key.name])
+
+        content = 'deb %s /\n' % self.kubic_repo_url()
+
+        logger.info('Installing repo file at %s...' % self.kubic_repo_path())
+        with open(self.kubic_repo_path(), 'w') as f:
+            f.write(content)
+
+    def rm_kubic_repo(self):
```

Comment:
shall we call this method after installing podman?

Reply:
we might want the repo later during `apt upgrade`, but maybe we should call this during the `rm-repo` command?

```
ubuntu@ubuntu:~$ sudo ~/cephadm rm-repo 
Removing repo GPG key /etc/apt/trusted.gpg.d/kubic.release.gpg...
Removing repo at /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list...
```